app/src/main/res/values/strings.xml

```diff
@@ -162,4 +149,27 @@
     <string name="copy_message_fetch">Copying message to clipboard</string>
     <string name="copy_failed">Copying to clipboard failed</string>
     <string name="try_again_error_msg">Something went wrong, try again</string>
+    <string name="cancel">Cancel</string>
+    <string name="no_message_to_display">Sorry, no messages to display!</string>
+    <string name="message_here">Message here</string>
+    <string name="sender_name">Sender name</string>
+    <string name="social">Social</string>
+    <string name="my_stream">MyStream</string>
+    <string name="subject">Subject</string>
+    <string name="crop">CROP</string>
+
+    <!--Placeholder strings-->
+    <string name="hint_person">Person</string>
+    <string name="username">Email</string>
+    <string name="password">Password</string>
+    <string name="tap_message">Tap on a message to compose reply</string>
+    <string name="hint_subject">Topic</string>
+    <string name="hint_stream">Stream</string>
+    <string name="search">Search</string>
+    <string name="hint_search_people">Search people</string>
+    <string name="hint_search_stream">Search stream</string>
+    <string name="topic">Topic</string>
+    <string name="server_url">Server URL</string>
```

Comment:
@abhaymaniyar you can use "find usages" option
For eg: to find usages of "server_url" string just right click on it and choose "find usages" to see this resource's project wide usages.